ui/packages/audio/src/Audio.svelte

```diff
@@ -29,13 +31,33 @@
 	// export let type: "normal" | "numpy" = "normal";
 
 	let recording = false;
-	let recorder: MediaRecorder;
+	let recorder: IMediaRecorder;
 	let mode = "";
-	let audio_chunks: Array<Blob> = [];
-	let audio_blob;
+	let header: Uint8Array | undefined = undefined;
+	let pending_stream: Array<Uint8Array> = [];
 	let player;
 	let inited = false;
 	let crop_values = [0, 100];
+	const STREAM_TIMESLICE = 500;
+	const NUM_HEADER_BYTES = 44;
+	let audio_chunks: Array<Blob> = [];
+	let audio_blob;
+	let module_promises:
+		| [
+				Promise<typeof import("extendable-media-recorder")>,
+				Promise<typeof import("extendable-media-recorder-wav-encoder")>
+		  ];
+
+	function get_modules() {
+		module_promises = [
+			import("extendable-media-recorder"),
+			import("extendable-media-recorder-wav-encoder")
+		];
+	}
+
+	if (streaming) {
+		get_modules();
+	}
```

Comment:
We only get the modules if we are streaming. The vanilla stuff is fine for recording.